### PR TITLE
(VDB-1606) Add Flip v1.1.0 storage transformers

### PR DESCRIPTION
**Pull request by rmulhol**
- include cat field

### CODE REVIEW

File: transformers/storage/bid_storage.go [hunk 1]
```diff
@@ -17,16 +17,18 @@
...
-	Ilk    = "ilk"
-	Gem    = "gem"
-	Beg    = "beg"
-	Pad    = "pad"
-	Ttl    = "ttl"
-	Tau    = "tau"
-	Kicks  = "kicks"
-	Live   = "live"
-	Vow    = "vow"
+	Beg   = "beg"
```
Comment (yaoandrew) [inline transformers/storage/bid_storage.go:20]: 🥇

**Review by yaoandrew — APPROVED** [review]
LGTM 🚀

File: environments/docker.toml [hunk 2]
```diff
@@ -11,22 +11,31 @@
...
+        "flip_eth_a_v1_1_0",
         "flip_knc_a_v1_0_8",
         "flip_knc_a_v1_0_9",
+        "flip_knc_a_v1_1_0",
         "flip_mana_a_v1_0_9",
+        "flip_mana_a_v1_1_0",
         "flip_sai_v1_0_0",
         "flip_tusd_a_v1_0_7",
         "flip_tusd_a_v1_0_9",
+        "flip_tusd_a_V1_1_0",
```
Comment (elizabethengelman) [inline environments/docker.toml]: super small, but I think we want this `V1` to be `v1` so that it matches with the exporter?

**Review by elizabethengelman — APPROVED** [review]
Looks good - just that one comment about the flip_tusd transformer name in docker.toml.

File: environments/docker.toml (hunk 2 above)
Comment (rmulhol) [reply environments/docker.toml]: Nice catch! 👍

**Pull request merged**